pkg/reconciler/pipelinerun/cancel.go

```diff
@@ -136,9 +141,15 @@ func cancelPipelineTaskRuns(ctx context.Context, logger *zap.SugaredLogger, pr *
 	return errs
 }
 
-// getChildObjectsFromPRStatus returns taskruns and runs in the PipelineRunStatus's ChildReferences or TaskRuns/Runs,
+// getChildObjectsFromPRStatus returns all taskruns and runs in the PipelineRunStatus's ChildReferences or TaskRuns/Runs,
 // based on the value of the embedded status flag.
 func getChildObjectsFromPRStatus(ctx context.Context, prs v1beta1.PipelineRunStatus) ([]string, []string, error) {
+	return getChildObjectsFromPRStatusForTaskNames(ctx, prs, sets.NewString())
+}
```

Comment:
is this function still used? can't seem to find it in use 🤔 

also it's a wrapper around `getChildObjectsFromPRStatusForTaskNames` so if there's any use, can we remove it and use `getChildObjectsFromPRStatusForTaskNames` directly?

Reply:
Good call - doing so now.